app/webpacker/css/admin_v3/dashboard/dashboard_item.scss

```diff
@@ -230,7 +230,7 @@ div.dashboard_item {
     }
 
     &:hover {
-      background-color: $spree-green;
+      background-color: $color-10;
```

Comment:
This is in the new design, so we don't need to use the `color-10` var here. 

(In fact, we don't need to define color-7..12 at all because they're not in the old design)

Reply:
Ok, not sure to follow you on this, I probably miss something. 

If I let as it, and therefor use `$spree-green` here's the result:
<img width="642" alt="Capture d’écran 2023-09-06 à 10 21 27" src="https://github.com/openfoodfoundation/openfoodnetwork/assets/296452/6896001d-7069-41e7-9e44-2f9fdf8dc45f">

Using the `$color-10`, it's the same color used for hovering a button (which is the expected result I guess). Saying that, I should used `$color-btn-hover-bg` to be more comprehensive/readable. Will change that.

Reply:
Hi @jibees , sorry I wasn't very clear there. I meant instead of using `$color-10`, we can use `$orient`, which is a bit easier to read (although I admit "orient" doesn't really describe a colour anyway).

Using `$color-btn-hover-bg` sounds like a great idea, it should help us manage the colour for all buttons..